Update docs for Algolia 2.1.5 changes. (#6063)
umbraco/UmbracoDocs: marketplace-and-integrations/integrations/algolia.md

    .Build();
```

> ### Version 2.1.5 and up

Reviewer: move this section above ### Version 2.0.0 to 2.1.5, as the newest are documented first. So the order would be: Version 2.0.0 to 2.1.5 **then** Version 2.0.0 to 2.1.5 **then** Up to version 1.5.0

> Following [this](https://github.com/umbraco/Umbraco.Cms.Integrations/issues/188) issue, that affects `Umbraco.TinyMCE` property editor in Umbraco 13, we have passed the `IProperty` object to the parse method of the converters.
> 
> A custom converter will look like this, allowing developers to add their implementation based on the content property.

Reviewer: ```suggestion
As a resolution for [an issue](https://github.com/umbraco/Umbraco.Cms.Integrations/issues/188) that affects `Umbraco.TinyMCE` property editor in Umbraco 13, the `IProperty` object has been passed to the parse method of the converters.

A custom converter will look like this, allowing developers to add their implementation based on the content property:
```
